src/recipe/Baked.Recipe.Service.Application/CodingStyle/CommandPattern/CommandPatternCodingStyleFeature.cs

```diff
@@ -55,7 +56,12 @@ public void Configure(LayerConfigurator configurator)
 
         configurator.ConfigureSwaggerGenOptions(swaggerGenOptions =>
         {
-            swaggerGenOptions.OperationFilter<XmlExamplesFromClassOperationFilter>(_methodNames, configurator.Context.GetDomainModel());
+            var fileProvider = configurator.Context.GetGeneratedFileProvider();
+            using (var file = new FileStream(fileProvider["RequestResponseExamples"], FileMode.Open))
+            {
+                var methodExamplesDictionary = JsonConvert.DeserializeObject<Dictionary<string, RequestResponseExampleData>>(new StreamReader(file).ReadToEnd()) ?? [];
+                swaggerGenOptions.OperationFilter<XmlExamplesFromClassOperationFilter>(_methodNames, methodExamplesDictionary);
+            }
```

Comment:
you may move these read operations to generated file provider. this way i see a lot of file stream and read codes everywhere. try to reduce file read/write code duplication